jobs/Neatline_Job_DuplicateRecords.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @package     omeka
+ * @subpackage  neatline
+ * @copyright   2014 Rector and Board of Visitors, University of Virginia
+ * @license     http://www.apache.org/licenses/LICENSE-2.0.html
+ */
+
+
+class Neatline_Job_DuplicateRecords extends Omeka_Job_AbstractJob
```

Comment:
Thanks, both! The editing checkpoint use case is a very helpful one to keep in mind.